pinot-common/src/main/java/org/apache/pinot/common/config/TableConfig.java

```diff
@@ -82,7 +88,8 @@ public TableConfig() {
   private TableConfig(String tableName, TableType tableType, SegmentsValidationAndRetentionConfig validationConfig,
       TenantConfig tenantConfig, IndexingConfig indexingConfig, TableCustomConfig customConfig,
       @Nullable QuotaConfig quotaConfig, @Nullable TableTaskConfig taskConfig, @Nullable RoutingConfig routingConfig,
-      @Nullable Map<InstancePartitionsType, InstanceAssignmentConfig> instanceAssignmentConfigMap) {
+      @Nullable Map<InstancePartitionsType, InstanceAssignmentConfig> instanceAssignmentConfigMap,
+      @Null List<FieldConfig> fieldConfigList) {
```

Comment:
what is @ Null, is it suppose to be @ Nullable?

Reply:
Yes that was a typo.